### PR TITLE
add seiqrdp-model

### PR DESCRIPTION
**Pull request by lesezhu**
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by Taha-Rouabah** [comment]
I agree to be maintainer for the conda recipe

**Comment by Belaloui** [comment]
I agree on being a maintainer!

**Comment by Abduhu** [comment]
I agree on being a maintainer too!

File: recipes/seiqrdp-model/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+package:
```
Comment (chrisburr) [inline recipes/seiqrdp-model/meta.yaml:1]:
```suggestion
{% set version = "7.5.3" %}

package:
```

File: recipes/seiqrdp-model/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  name: seiqrdp-model
+  version: 7.5.3
```
Comment (chrisburr) [inline recipes/seiqrdp-model/meta.yaml:3]:
```suggestion
  version: "{{ version }}"
```

File: recipes/seiqrdp-model/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+source:
+  url: https://pypi.io/packages/source/s/seiqrdp-model/seiqrdp_model-7.5.3.tar.gz
```
Comment (chrisburr) [inline recipes/seiqrdp-model/meta.yaml:6]: It's better to use a version variable to make sure the version and sources are kept in sync. It also makes it possible to the bot to do automatic update PRs:
```suggestion
  url: https://pypi.io/packages/source/s/seiqrdp-model/seiqrdp_model-{{ version }}.tar.gz
```

File: recipes/seiqrdp-model/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - colorama
+    - matplotlib
```
Comment (chrisburr) [inline recipes/seiqrdp-model/meta.yaml:21]: Does this need the `qt` frontend for matplotlib? If not:
```suggestion
    - matplotlib-base
```

**Pull request merged**